src/plugins/duplicate-comments/index.js

```diff
@@ -34,6 +34,7 @@ function getCommentHash(comment) {
     const startIdx = comment.indexOf("[//]: # (") + 9;
     const endIndex = comment.indexOf(")", startIdx);
 
+    // eslint-disable-next-line
     return comment.substring(startIdx, endIndex);
```

Comment:
Could this be converted to `slice` rather than disabling the line?

Reply:
yeah, i was just trying to work through stuff.